opentelemetry-appender-tracing/benches/logs.rs

```diff
@@ -10,7 +10,7 @@
     | noop_layer_disabled         | 12 ns       |
     | noop_layer_enabled          | 25 ns       |
     | ot_layer_disabled           | 19 ns       |
-    | ot_layer_enabled            | 203 ns      |
+    | ot_layer_enabled            | 196 ns      |
```

Comment:
not related to this PR (as the processor is noop). This was consistent, so updated here.

Reply:
reverted, so no longer relevant. Pls ignore.